Formula/dartsim.rb

```diff
@@ -1,10 +1,9 @@
 class Dartsim < Formula
   desc "Dynamic Animation and Robotics Toolkit"
   homepage "https://dartsim.github.io/"
-  url "https://github.com/dartsim/dart/archive/v6.9.5.tar.gz"
-  sha256 "624c00b65e3a753cba50de038620860c86e2ac47b1793ae51f9427a4bcb14c32"
+  url "https://github.com/dartsim/dart/archive/v6.10.1.tar.gz"
+  sha256 "bd65b600e0983a5bdc0788d90dff0ba8fba8494d3f4af590ba1e81f44323eb4e"
```

Comment:
The tag `v6.10.1` was recently changed. The new sha256 for the new tag is `bf19cdef8e28dbc4059dcbb11997576d6e1d825791bd756e8272d2ddc5b147ce`

Reply:
👍